exec/exec.go

```diff
@@ -79,20 +80,23 @@ type VMContext interface {
 
 	CreateNewActor(addr address.Address, code cid.Cid, initalizationParams interface{}) error
 
-	// TODO: Remove these when Storage above is completely implemented
-	ReadStorage() ([]byte, error)
-	WriteStorage(interface{}) error
+	Verifier() Verifier
```

Comment:
I'm on the fence about whether we should flatten the VMContext API entirely or not. In some ways a flat API is easier to fake/mock/satisfy